Orange/widgets/data/owselectrows.py

```diff
@@ -492,7 +490,7 @@ def set_data(self, data):
         else:
             self.add_row()
 
-        self.update_info(data, self.data_in_variables, "In: ")
+        self.update_input_summary(data)
```

Comment:
If you place this line at the beginning of the method, you can remove upper `self.update_input_summary(None)`.